shared-libs/transitions/src/transitions/accept_case_reports.js

```diff
@@ -81,13 +79,21 @@ const updatePlaceUuid = (doc, registrations) => {
 };
 
 module.exports = {
+  name: 'accept_case_reports',
+  deprecated: false,
+  deprecatedIn: '',
+  getDeprecationMessage: () => {
```

Comment:
I think we should just skip adding these properties to non-deprecated transitions altogether and null-check them when they're used. (also applies to all transitions).

Reply:
@dianabarsan would you like also to remove the getDeprecationMessage to the non-deprecated transitions or leave it there?

Reply:
yes, please. remove all deprecation-like properties from non-deprecated transitions.

Reply:
Done!